src/inc/msquic.h

```diff
@@ -91,11 +91,12 @@ typedef enum QUIC_CREDENTIAL_TYPE {
 } QUIC_CREDENTIAL_TYPE;
 
 typedef enum QUIC_CREDENTIAL_FLAGS {
-    QUIC_CREDENTIAL_FLAG_NONE                       = 0x00000000,
-    QUIC_CREDENTIAL_FLAG_CLIENT                     = 0x00000001, // Lack of client flag indicates server.
-    QUIC_CREDENTIAL_FLAG_LOAD_ASYNCHRONOUS          = 0x00000002,
-    QUIC_CREDENTIAL_FLAG_NO_CERTIFICATE_VALIDATION  = 0x00000004,
-    QUIC_CREDENTIAL_FLAG_ENABLE_OCSP                = 0x00000008
+    QUIC_CREDENTIAL_FLAG_NONE                           = 0x00000000,
+    QUIC_CREDENTIAL_FLAG_CLIENT                         = 0x00000001, // Lack of client flag indicates server.
+    QUIC_CREDENTIAL_FLAG_LOAD_ASYNCHRONOUS              = 0x00000002,
+    QUIC_CREDENTIAL_FLAG_NO_CERTIFICATE_VALIDATION      = 0x00000004,
+    QUIC_CREDENTIAL_FLAG_ENABLE_OCSP                    = 0x00000008, // Schannel only currently
+    QUIC_CREDENTIAL_FLAG_DEFER_CERTIFICATE_VALIDATION   = 0x00000010, // Schannel only currently
```

Comment:
We should be consistent about trailing commas, especially in the public headers. The standards for both C and C++ allow both, so I don't really care which one, but we should be consistent. I like having them, because it makes diffs adding more values cleaner.

Reply:
Agree having them is nicer, that's why I put it there.

Reply:
I agree. So we should make an issue to fixup the other uses, at least in the public headers for now, and more potentially later. Likely not worth doing in this PR.

Reply:
I'll just do it here. When everything else is signed off, I'll push that change and merge.